examples/07_advanced/plot_beta_series.py

```diff
@@ -0,0 +1,382 @@
+fig, axes = plt.subplots(ncols=3, figsize=(20, 10))
+for i_trial in range(3):
+    plotting.plot_design_matrix(
+        lss_design_matrices[i_trial],
+        ax=axes[i_trial],
+    )
+    axes[i_trial].set_title(f'Trial {i_trial + 1}')
+
+fig.show()
+
+##############################################################################
+# Compare the three modeling approaches
+# -------------------------------------
+
+fig, axes = plt.subplots(
+    ncols=3,
+    figsize=(20, 10),
+    gridspec_kw={'width_ratios': [1, 2, 1]},
+)
+
+plotting.plot_design_matrix(
+    standard_glm.design_matrices_[0],
+    ax=axes[0],
+)
+axes[0].set_title('Standard GLM')
+
+plotting.plot_design_matrix(
+    lsa_glm.design_matrices_[0],
+    ax=axes[1],
+)
+axes[1].set_title('LSA Model')
+
+plotting.plot_design_matrix(
+    lss_design_matrices[0],
+    ax=axes[2],
+)
+axes[2].set_title('LSS Model (Trial 1)')
+
+fig.show()
+
+##############################################################################
+# Applications of beta series
+# ---------------------------
+# Beta series can be used much like resting-state data, though generally with
+# vastly reduced degrees of freedom than a typical resting-state run, given
+# that the number of trials should always be less than the number of volumes
+# in a functional MRI run.
+#
+# Two common applications of beta series are to functional connectivity and
+# decoding analyses.
+# For an example of a beta series applied to decoding, see
+# :ref:`sphx_glr_auto_examples_02_decoding_plot_haxby_glm_decoding.py`.
+# Here, we show how the beta series can be applied to functional connectivity
+# analysis.
+# In the following section, we perform a quick task-based functional
+# connectivity analysis of each of the two task conditions
+# ('language' and 'string'), using the LSS beta series.
+# This section is based on
+# :ref:`sphx_glr_auto_examples_03_connectivity_plot_seed_to_voxel_correlation.py`,
+# which goes into more detail about seed-to-voxel functional connectivity
+# analyses.
+import numpy as np
+
+from nilearn.maskers import NiftiSpheresMasker, NiftiMasker
+
+# Coordinate taken from Neurosynth's 'language' meta-analysis
+coords = [(-54, -42, 3)]
+
+# Initialize maskers for the seed and the rest of the brain
+seed_masker = NiftiSpheresMasker(
+    coords,
+    radius=8,
+    detrend=True,
+    standardize=True,
+    low_pass=None,
+    high_pass=None,
+    t_r=None,
+    memory='nilearn_cache',
+    memory_level=1,
+    verbose=0,
+)
+
+brain_masker = NiftiMasker(
+    smoothing_fwhm=6,
+    detrend=True,
+    standardize=True,
+    low_pass=None,
+    high_pass=None,
+    t_r=None,
+    memory='nilearn_cache',
+    memory_level=1,
+    verbose=0,
+)
+
+# Perform the seed-to-voxel correlation for the LSS 'language' beta series
+lang_seed_beta_series = seed_masker.fit_transform(lss_beta_maps['language'])
+lang_beta_series = brain_masker.fit_transform(lss_beta_maps['language'])
+lang_corrs = np.dot(
```

Comment:
Why not use a ConnectivityMeasure of Nilearn ?
It would avoid the np import.

Reply:
I just used the seed-to-voxel example's code for this, but I could use ConnectivityMeasure if that will make this example simpler. I haven't used ConnectivityMeasure in the past, though, and all of the examples seem to be for ROI-to-ROI analyses, where every column in an array is compared to every other column. That doesn't seem to fit easily with the seed-to-voxel approach. Am I misunderstanding the class and/or its examples?

Reply:
Indeed, you're right. I did not realize this when reading the code.